utils/src/chunked_evaluations.rs

```diff
@@ -0,0 +1,17 @@
+//! This module contains a type [ChunkedEvaluations],
+
+use serde::{Deserialize, Serialize};
+use serde_with::serde_as;
+
+/// This struct contains multiple chunk evaluations.
+#[serde_as]
+#[derive(Clone, Serialize, Deserialize)]
+#[serde(bound = "F: ark_serialize::CanonicalDeserialize + ark_serialize::CanonicalSerialize")]
```

Comment:
I don't think you need this bound no?

Reply:
Um, I think that bound restricts the possible types of `F`.
My first attempt was like 
```
pub struct ChunkedEvaluations<F: ark_ff::Field> {
...
```
I think we can restrict the `F` type bound by using above `serde(bound)`

Reply:
oh right, I think you should just add `pub struct ChunkedEvaluations<F> where F: PrimeField { ...` which is simpler : o

Reply:
Done https://github.com/o1-labs/proof-systems/pull/840/commits/1de06fb10d51ac2bfffddd1a622938ab704079b4